ComposureWatch/src/Components/RankIcon.jsx

```diff
@@ -38,11 +39,7 @@ const RankIcon = ({ rank }) => {
     },
   ];
 
-  let charRank;
-  let result = rankList.find((item) => item.id === rank);
-  if (result) {
-    charRank = result.img;
-  }
```

Comment:
If you did want to shorten up this code you could do something like:
```
const rankImg = rankList.find(({ id }) => id === rank)?.img;
```

this uses destructuring and [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)